README.md

```diff
@@ -1,6 +1,8 @@
 # OpenShift Release Tools
 
-This repository contains the process tooling for OpenShift releases.
+This repository contains the process tooling for OpenShift releases, implemented
+with an [OpenShift cluster](https://api.ci.openshift.org/console/), plus a
+Jenkins instance.
```

Comment:
In reality, we dispatch jobs to three different Jenkins masters today but whatever.